docs/introduction/infrastructure.md

```diff
@@ -1,161 +1,104 @@
 # Computational Infrastructure
 
-## Introduction
-One way of understanding the Inductiva API is to see it as an abstraction layer that
-allows you to use computational resources made available by different providers in a
-seamless and uniform fashion. 
+One way to understand the Inductiva API is to see it as an intermediary between 
```

Comment:
This and the previous sentence we had, seem to be afraid to tell transparently that this is exactly what we are. Could we do that and go straight to the point. We can keep the intermediary and even had a mention to simulators or computational loads.